packages/components/src/components/hds/icon/index.ts

```diff
@@ -44,6 +56,14 @@ export default class HdsIcon extends Component<HdsIconSignature> {
     }
   }
 
+  get name(): string | undefined {
```

Comment:
[thought] if we change the name, we will break tests when the "carbon" theme is turned on, and replacing the name across all the tests would be quite an amount of work